setup.py

```diff
@@ -1,207 +1,3 @@
-    # Any package you put here will be installed by pip when your project is
-    # installed, so they must be valid existing projects.
-    #
-    # For an analysis of "install_requires" vs pip's requirements files see:
-    # https://packaging.python.org/en/latest/requirements.html
-    # install_requires=['peppercorn'],  # Optional
-
-    # List additional groups of dependencies here (e.g. development
-    # dependencies). Users will be able to install these using the "extras"
-    # syntax, for example:
-    #
-    #   $ pip install sampleproject[dev]
-    #
-    # Similar to `install_requires` above, these must be valid existing
-    # projects.
-    # extras_require={  # Optional
-    #     'dev': ['check-manifest'],
-    #     'test': ['coverage'],
-    # },
-
-    # If there are data files included in your packages that need to be
-    # installed, specify them here.
-    #
-    # If using Python 2.6 or earlier, then these have to be included in
-    # MANIFEST.in as well.
-    # package_data={  # Optional
-    #     'sample': ['package_data.dat'],
-    # },
-
-    # Although 'package_data' is the preferred approach, in some case you may
-    # need to place data files outside of your packages. See:
-    # http://docs.python.org/3.4/distutils/setupscript.html#installing-additional-files
-    #
-    # In this case, 'data_file' will be installed into '<sys.prefix>/my_data'
-    # data_files=[('my_data', ['data/data_file'])],  # Optional
-
-    # To provide executable scripts, use entry points in preference to the
-    # "scripts" keyword. Entry points provide cross-platform support and allow
-    # `pip` to create the appropriate form of executable for the target
-    # platform.
-    #
-    # For example, the following would provide a command called `sample` which
-    # executes the function `main` from this package when invoked:
-    # entry_points={  # Optional
-    #     'console_scripts': [
-    #         'sample=sample:main',
-    #     ],
-    # },
-    platforms='any',
-
-    # List additional URLs that are relevant to your project as a dict.
-    #
-    # This field corresponds to the "Project-URL" metadata fields:
-    # https://packaging.python.org/specifications/core-metadata/#project-url-multiple-use
-    #
-    # Examples listed include a pattern for specifying where the package tracks
-    # issues, where the source is hosted, where to say thanks to the package
-    # maintainers, and where to support the project financially. The key is
-    # what's used to render the link text on PyPI.
-    project_urls={  # Optional
-        'Documentation': 'https://mlfinlab.readthedocs.io/en/latest/',
-        'Bug Reports': 'https://github.com/hudson-and-thames/mlfinlab/issues',
-        'Project Boards': 'https://github.com/orgs/hudson-and-thames/projects',
-        'Source': 'https://github.com/hudson-and-thames/mlfinlab',
-        'Blog': 'https://hudsonthames.org/blog/',
-    },
-
-    # Create package
```

Comment:
We need to please keep these comments:
```
    # Create package
    # python setup.py bdist_wheel
    # python3 -m twine upload --repository-url https://test.pypi.org/legacy/ dist/*  (This is the test repo)
    # twine upload dist/*  (This is official repo)
```
It helps me to push the python package to PyPi index.

Reply:
It's unnecessary comments, so I'm not sure that this is a great idea to keep it within a package. I think it'd be better to keep it somewhere locally.